src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -206,6 +206,7 @@ export type Project = {
   latestDeploys: Record<string, Pick<Deploy, 'dateFinished' | 'version'>> | null;
   builtinSymbolSources?: string[];
   stats?: Array<[number, number]>;
+  transactionStats?: Array<[number, number]>;
```

Comment:
Would it be useful to name this tuple?

Reply:
Done. I also consolidated the new type with other similarly shaped types.